imjoy/worker_template.py

```diff
@@ -550,7 +545,6 @@ def main():
     parser = argparse.ArgumentParser()
     parser.add_argument("--id", type=str, required=True, help="plugin id")
     parser.add_argument("--secret", type=str, required=True, help="plugin secret")
-    parser.add_argument("--namespace", type=str, default="/", help="socketio namespace")
```

Comment:
Is it ok to remove this option for the plugin connection? We don't use it.

Reply:
Yes, we can remove it.

Reply:
Thanks!